_layouts/page.html

```diff
@@ -13,6 +13,7 @@
       </div>
       <div class="tablet:grid-col-8 tablet:grid-offset-1">
         <h1>{{page.title}}</h1>
+        {% include alert-proto.html %}
```

Comment:
Could we move this below the `<div class="crt-landing--separator"></div>` for the time being so the heading and decorative accent stay together?

Reply:
Good catch. I'll push another commit.